### ISSUE TITLE
Peer Review -- midterm

### ISSUE THREAD

**Issue by kel89** [gregstepaniouk/MLBigDataProject#6]
I think you are off to a very promising start. From what I can tell, you have done a lot of the difficult leg work: you have cleaned and aggregated most of the data, thought about how you want to model it, and done some preliminary analysis. What I like most about what you have done is that you have put some thought into what you *expect* to come out of your analysis. You looked at some histograms and scatter plots based on what logically you would expect. This is setting you up for some good hypothesis testing that I feel makes for good analysis. 

You did a good job describing what feature transformations you did, and gave some reasons for why you did them, but I am not totally convinced they are legitimate. For example, you aggregate a lot of the data to deal with missing values. I do not doubt that this is reasonable, but you do not offer much of an explanation as to why you are justified in summing some variables, and taking weighted averages of others. Another point that confused me a bit was how you choose to use rank to gather the data. I wonder if by choosing those four stocks, and then by only looking at top broker deals you are introducing bias into your model? Would it be reasonable to think that large broker deals offer better rates than smaller ones, especially on the blue chip names you used to rank? I am not sure there is much you can do about this (if it is even actually a problem), but it may be worth mentioning, or getting a test set that does not reflect this (if possible). On that note, there is no talk of what (if any) you plan to use as a test/validation set. I think it would be nice to give a bit of an explanation of what you are using to train versus test the model. 

In terms of the models you are running. I like the linear regression, and the conclusions you draw from it, however in the table of coefficients it would be nice to also present the p-value, t-stat, or standard error so the reader could judge the significance of the coefficients. Lastly, you seem to be very worried about overfitting, but you say you have ~1.5 millions observations, and for your linear mode at least you are only considering 10 or so predictors, so I would be much more worried about under-fitting. It seems to me you have tremendous latitude to introduce other features, for example, it might be interesting to use some highly non-linear polynomial terms. 

Overall, I think you are off to a great start. Good luck!
